hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/authentication/GatewaySecurityTokenAuthenticationConfigurationItem.java

```diff
@@ -58,7 +58,7 @@ public GatewaySecurityTokenAuthenticationConfigurationItem(
         detailLayout = new VerticalLayout();
         detailLayout.setImmediate(true);
 
-        final Button gatewaytokenBtn = SPUIComponentProvider.getButton("TODO-ID", "Regenerate Key", "",
+        final Button gatewaytokenBtn = SPUIComponentProvider.getButton("TODO-ID", i18n.getMessage("configuration.button.regenerateKey"), "",
```

Comment:
Check if "TODO-ID" can be removed. Setting it to null should result in a button without an ID